### ISSUE TITLE
Issue in Article CSS (Issue from Real World Issues)

### ISSUE THREAD

**Issue by Cameron-C-Chapman** [GoThinkster/conduit-bootstrap-template#1]
https://github.com/gothinkster/realworld/issues/127

This issue was posted in the RealWorld issues repo but I think it would need to be addressed here if we determine there is an issue.

**Comment by cirosantilli** [comment]
Yes, it seems that it is because this repo is out of sync with https://demo.productionready.io/main.css posted screenshots at: https://github.com/gothinkster/conduit-bootstrap-template/issues/5#issuecomment-829104220
